src/help/help_topic_generators.cpp

```diff
@@ -309,20 +309,8 @@ std::string unit_topic_generator::operator()() const {
 	const int screen_width = video::game_canvas_size().x;
 
 	ss << _("Level") << " " << type_.level();
-	ss << "\n\n";
-
-	ss << "<img>src='" << male_type.image();
-	ss << "~RC(" << male_type.flag_rgb() << ">red)";
-	if (screen_width >= 1200) ss << "~SCALE_SHARP(200%,200%)";
-	ss << "' box='no'</img> ";
-
-	if (female_type.image() != male_type.image()) {
-		ss << "<img>src='" << female_type.image();
-		ss << "~RC(" << female_type.flag_rgb() << ">red)";
-		if (screen_width >= 1200) ss << "~SCALE_SHARP(200%,200%)";
-		ss << "' box='no'</img> ";
-	}
 
+	// Portraits
```

Comment:
Why are the unit sprite images above removed?

Reply:
If I put the portraits after the inline images, the top of the floating image will be anchored to the bottom border of the inline images instead of the top border.